lib/taurus/core/util/enumeration.py

```diff
@@ -95,9 +97,9 @@ def __init__(self, name, enumList, flaggable=False, no_doc=False):
                     raise EnumException(
                         "flagable enum does not accept tuple items")
                 x, i = x
-                if not isinstance(x, (str, unicode)):
+                if not isinstance(x, (str, str)):
```

Comment:
All these checks of `isinstance(x, (str,str))` and `isinstance(ix, (nt,int))` can now be simplified

Reply:
I suggest to use future.utils.string_type and future.utils.integer_types for type checking.
It allows us to keep code compatible with python2 and python3 .